### ISSUE TITLE
MPDroid not on Play Store

### ISSUE THREAD

**Issue by maxrabin** [abarisain/dmix#850]
https://play.google.com/store/apps/details?id=com.namelessdev.mpdroid is now giving a 404 Not Found error. Where'd it go? Why had it been delisted?

**Comment by abarisain** [comment]
Yeah, Google removed it as they require me to host a privacy policy somewhere because of mpdroid's permissions


Annoying

**Comment by abarisain** [comment]
I'll try to fix the issue in the coming weeks

**Comment by robbat2** [comment]
Throw it up on the wiki pages of this repo?

**Comment by abarisain** [comment]
Good idea. I just hope that i don't have to add a link inside the app.

Oh and I have to actually write it. It's pretty straightforward though: MPDroid uses crashlytics, and that's it. I don't collect anything.

**Comment by ranperry** [comment]
Nothing inside the app. You just need to refer to a URL.  Take a look here:

https://developers.google.com/actions/policies/privacy-policy-guide

And:

https://www.iubenda.com/en?utm_source=adwords&utm_medium=ppc&utm_campaign=aw_general_canada_bmm&utm_term=%2Bprivacy%20%2Bpolicy&utm_content=226000081610&gclid=CjwKCAiApo3SBRA4EiwAty8i-qV1AhlHSL32DfBxVB_UCLRFAutNirz9QsyXxBf69PaK1xgJPdChKhoCvIUQAvD_BwE
​

**Comment by abarisain** [comment]
Guess I misunderstood the rules, thanks.

Glad to see people still using it in the store :)

**Comment by jeroenhabets** [comment]
Found it as I couldn't find MPDroid in the app store. Glad to find out this project is still alive...

**Comment by abarisain** [comment]
It's not really though :)

I totally forgot to put the app back though. I will

**Comment by ildar** [comment]
Malp is developed more or less
